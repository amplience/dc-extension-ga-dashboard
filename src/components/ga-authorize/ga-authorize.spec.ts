import { render } from '@testing-library/svelte';
import GAAuthorize from './ga-authorize.svelte';

xdescribe('No Indexes Placeholder', () => {
  it('should render the No Indexes Placeholder component', () => {
    const { container } = render(GAAuthorize, {});

    expect(container.firstChild).toMatchSnapshot();
  });
});
