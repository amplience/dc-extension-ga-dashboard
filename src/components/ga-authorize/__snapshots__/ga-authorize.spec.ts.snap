// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GAAuthorize should render the GAAuthorize component with auth button 1`] = `
<div>
  <div
    class="svelte-1i1pnuw"
  >
    <div
      class="ga-authorize svelte-1i1pnuw"
    >
      <div
        class="auth-button svelte-1i1pnuw"
        id="auth-button"
      />
    </div>
  </div>
</div>
`;
